src/components/DropList/DropList.Select.jsx

```diff
@@ -166,12 +161,36 @@ function Select({
     >
       <A11yTogglerUI {...getToggleButtonProps()}>Toggler</A11yTogglerUI>
       <MenuListUI
+        data-event-driver
         className={`${DROPLIST_MENULIST} MenuList-Select`}
         {...getMenuProps({
           onKeyDown: handleMenuKeyDown,
           onFocus: e => {
             onMenuFocus(e)
           },
+          onBlur: e => {
+            if (closeOnBlur) {
+              /**
+               * Closing on blur
+               *
+               * When clicking on the toggler, this event gets fired _before_
+               * so it sets the isOpen flag to false, and then the click event happens
+               * and sets isOpen to true, making the DropList never close on cliking
+               * the toggler.
+               *
+               * Here we wait a little bit to see if the next element gathering focus
+               * is indeed the toggler, and only close the DropList if it isn't
+               */
+              setTimeout(() => {
+                if (
+                  !document.activeElement.classList.contains('DropListToggler')
```

Comment:
If there is a custom toggler, would this class be added automatically, or when creating it, we must remember to add this class?

Reply:
gets added for you 👍